src/components/product-card/product-card.component.scss

```diff
@@ -0,0 +1,107 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 19vh;
+  padding: 2vw;
+  width: 7vw;
+  background-color: #fff;
+  border-radius: 0.5vw;
+  margin-right: 1vw;
+}
+
+.image-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.image-container img {
+  width: 90%;
+  height: 150%;
+}
+
+p {
+  margin: 4% 0;
+  font-family: 'Quicksand', serif;
+  font-weight: bold;
+}
+
+.add-button {
+  background: #3F51B5;
+  color: white;
+  width: 8vw;
+  padding: 0.5vw;
+  border-radius: 0.3vw;
+  border: none;
+  cursor: pointer;
+  font-family: 'Quicksand', serif;
+  font-weight: bold;
+}
+
+@media only screen and (max-width: 360px) {
+  .container {
+    height: 22vh;
+    padding: 1vw;
+    width: 25vw;
+    margin-left: -10vw;
+  }
+
+  p {
+    margin: 2% 0;
+    font-size: 3vw;
+  }
+
+  .add-button {
+    width: 20vw;
+    height: 7vw;
+    font-size: 2.5vw;
+  }
+
+  .image-container {
+    width: 90%;
+    height: 40%;
+    margin-top: 4vw;
+  }
+
+  .image-container img {
+    width: 90%;
+    height: 100%;
+
+  }
+}
+
+@media only screen and (min-width: 740px) and (max-width: 740px) {
+  .container {
+    height: 26vh;
+    padding: 1vw;
+    width: 11vw;
+    margin-left: 1vw;
+  }
+
+  p {
+    margin: -4% 0;
+    font-size: 1vw;
+  }
+
+  button {
+    width: 10vw;
+    height: 3vw;
+    font-size: 1vw;
+  }
+  .image-container {
```

Comment:
because you need three:
.image-container
.image-container img

Reply:
These are media queries for the resolutions we handle.